### PR TITLE
Register my domain

### PR DESCRIPTION
**Pull request by shoaib-shaikhh**

### CODE REVIEW

**Comment by Tweak4141** [comment]
@shoaib-shaikhh I am one of the maintainers here. Please let your company know that I have sent an email to them. Additionally, please tell them to reply back to me at tweak@is-a-good.dev

File: sub-logs/shoaib-shaikhh.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "owner": {
+    "username": "shoaib-shaikhh",
+    "email": "shoaib-shaikhh@acquaintsofttech.com"
+  },
+  "target": {
+    "CNAME": {
+      "name": "shoaib-shaikhh",
+      "value": "shoaib-shaikhh-dev.github.io"
```
Comment (DEV-DIBSTER) [inline sub-logs/shoaib-shaikhh.json:9]: You need a valid CNAME record. Hit with a 404.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

File: sub-logs/shoaib-shaikhh.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+  "owner": {
+    "username": "shoaib-shaikhh",
+    "email": "shoaib-shaikhh@acquaintsofttech.com"
+  },
+  "target": {
+    "CNAME": {
+      "name": "shoaib-shaikhh",
+      "value": "https://shoaib-shaikhh.is-a.dev/shoaib-shaikhh-dev.github.io/"
```
Comment (DEV-DIBSTER) [inline sub-logs/shoaib-shaikhh.json:9]: Invalid CNAME records. Please see link below:

https://support.dnsimple.com/articles/cname-record/#whats-a-cname-record

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

**Review by DEV-DIBSTER — APPROVED** [review]

**Comment by wclarkey** [comment]
Before this is merged, we require you to star the repository

**Review by Tweak4141 — APPROVED** [review]

**Pull request merged**